package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pangolindex/components",
-  "version": "2.0.3",
+  "version": "2.0.4",
```

Comment:
we dont need to update version manually, it will be automatically updated via github actions

Reply:
ok ✅